src/templates/es/base_chapter.html

```diff
@@ -1,6 +1,8 @@
 {% extends "base/base_chapter.html" %}
 
 {% block title %}{{ metadata.get('title') }} | {{ year }} | Web Almanac por HTTP Archive{% endblock %}
+{% block citation_title %}Web Almanac {{ year }}: Capítulo {{ metadata.get('title') }}{% endblock %}
```

Comment:
If this is English only, could we revert the translations?

Reply:
I'd rather keep them for consistency, for a number of reasons:
- We may use them in the future, so having them (and have new translators add these for new languages, rather than me having to back guess them like I've had to do here) would facilitate that. Particularly if we want to expose the citation details on the site itself as discussed in #1325 
- We have a script to check translations have the same number of lines to ensure lines aren't accidentally missed, so at very least we'd need to add blank lines for this so that doesn't faiul. And if going that far, then might as well just have the translations.
- Lack of meta data could cause a foreign-language page to fail if the templates ever change and most testing is done in English (though this _should_ be picked up by the automated testing).

As much as possible I try to keep the translations the same as English. I think once we start down the path of only having certain pieces in English it's dangerous territory. I accept we have that to a point with the `{% if lang == "en" %}` block in this PR but would rather leave it at that and still have the other translations.